app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<%@users.each do |user|%>
+ <%= link_to user_path(user) do%>
+ <%= render 'shared/user', user: user%>
+ <%end%>
+<%end%>
```

Comment:
- [ ] @SabaAhmad404  Great work on your pages, you have added all pages required, however, the designsoin all pages doesnt match the design given in the wireframe, below is a comparison of some  of the pages:
Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![](https://...Dark.png)  |  ![](https://...Ocean.png)